src/daria/image/image.py

```diff
@@ -112,25 +147,21 @@ def __init__(
                 "Invalid image data. Provide either a path to an image or an image array."
             )
 
-        # Read metadata from file or create from input arguments
-        if read_metadata_from_file:
-            self.create_metadata_from_file(metadata_path)
-        else:
-            self.origo = origo if isinstance(origo, np.ndarray) else np.array(origo)
-            self.width = width
-            self.height = height
-            self.dim = dim
-        self.shape = self.img.shape
+        if curvature_correction is not None:
+            self.img = curvature_correction(self.img)
+            self.width = curvature_correction.config["crop"]["width"]
+            self.height = curvature_correction.config["crop"]["height"]
```

Comment:
Maybe it would be better, if we introduce `width` and `height` as attributes of `CurvatureCorrection`? The benefit would be that the in some sense hardcoded addressing of the navigation through the config of `CurvatureCorrection` would not be required. And it would also not be seen to the outside of `CurvatureCorrection`, which would mean the above suggestion would result in a slightly more robustness. Instead of using `width` as name for the attribute, maybe `target_width` would be a better choice, as this property is connected to the corrected image and not `CurvatureCorrection`. And as object itself, `CurvatureCorrection` does not see any image before the user does not provide it.

Reply:
I agree that there might possibilities for improvement here. I will however not touch it now (as you said not to touch the curvature.py, and I think that the current solution is ok)